### DISCUSSION TITLE
Wake up not working recently?

### DISCUSSION THREAD

**Discussion by jbanyer** [timdorr/tesla-api#446] [Q&A]
Just recently (Sep 2021) I've noticed my Model 3 won't wake up reliably, either from the Tesla app or via the API `wake_up` command. I've noticed other people making similar comments online. Does anyone have any insights?

**Comment by BlackPhoenix** [comment]
@jbanyer the wakeup call sends a SMS to the car (from Tesla's servers in California). If the car doesn't have cell reception, it will never receive it. Being on wifi is irrelevant - SMS is always via cell signal.

**Reply by jbanyer** [reply]
Interesting to know, thanks. I don't think that's my problem, as the car is parked in a location with good cell reception. Today the wake up worked fine first go with the car in the same place where it was failing - so it appears to be intermittent. I only raised this question now because I've notice quite a few other people reporting this issue and wondered if there was a systematic problem. Maybe Tesla is (or has been) having problems with it.

**Reply by rawmean** [reply]
one of our cars (a model S) refuses to wake up. Rebooting the car fixes the problem but this is the second day that I need to reboot the car.
